content/blog/open-sourcing-packetrusher-a-5g-core-performance-tester.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Open-sourcing PacketRusher: A 5G Core performance tester"
```

Comment:
Hi there. I'm your friendly neighborhood HPE Developer blog post editor. I'll be editing your post mostly for grammar, HPE Brand compliance, and readability.

One thing I want to point out from the start is that HPE Brand requests that we use "sentence case" in anything we write, whether it's blog posts, presentations, web pages, etc.  This means that, as opposed the traditional "title case", where you capitalize the initial letter in each word that might be important in a title (example: How to Bake a Cake), instead you only initially capitalize the first word in a sentence, headline or title. The two exceptions to this rule are when the word happens to be a proper noun (i.e. the name or title of something (example: Microsoft Word) OR it's as shown on a screen capture (example: Click Here - if it's actually shown on the screen that way in your demo).

This means, as an editor, I need to learn about all the different proper names in technology. So I go around googling a variety of items that I'm not familiar with. In your case, it's 5G Core. This one is problematic in that, on some pages it's shown as "5G Core" and on others it's shown as "5G core". 

So... as I am not the subject matter expert on this topic, I'm going to leave it to you to decide which way it should be based on the guidance I've given you above :-). I hope that helps.

Reply:
Hi @drensing! Thanks a lot for checking that!! And well, the answer to this question is not easy. I think why you observed both "5G Core" and "5G core" is that we can refer to the "5G Core" both with a common noun and a proper noun.
A bit like if "word" were to be both the common noun for a word processor and "Word" the proper noun of a given word processors. I think it's pretty similar here, "5G Core" refers to the central entity to a 5G network, while "5G core" basically mean the core of the 5G network which is of course the "5G Core".
So TL;DR, I would say it's a bit like history that can be used either as a proper noun and a common noun, but I'm not a linguist and I'm far from being a native English-speaker! Thanks for the thoughts experiments in any cases :D